patches/server/0400-Implement-Player-Client-Options-API.patch

```diff
@@ -119,6 +119,10 @@ index b04aaab4f7cb7367d0fbc6268b0db269b55b2d17..3f25e799ba13d8280c644a943ca0d191
 +            return type.getType().cast(getMainHand());
 +        } else if(com.destroystokyo.paper.ClientOption.VIEW_DISTANCE.equals(type)) {
```

Comment:
might as well fix the missing spaces on the other if statements here. 
And idk if it makes literally any difference, but is there a reason not to use reference equality here?